device-types/HPE/Aruba-EC-L-B.yaml

```diff
@@ -1,24 +1,24 @@
 ---
 manufacturer: HPE
-model: Aruba ec-l-b
+model: Aruba EC-L-B
 slug: hpe-aruba-ec-l-b
 part_number: '201270'
 u_height: 1
 is_full_depth: true
 airflow: front-to-rear
+weight: 36.0
+weight_unit: lb
 console-ports:
   - name: console
     type: de-9
     label: console
 power-ports:
```

Comment:
Hi Thank you for your contribution.
Are these modular PSUs? If so they should be modeled as `module-bays` with the required PSUs added to the modules folder if they don't already exist.